### ISSUE TITLE
Install Qt Creator and SFML

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/djog_unos_2018#4]
There is a Qt Creator project in this GitHub. If you can get it running, mention that here. If you have trouble to do so, mention that here.

[**edit by @richelbilderbeek: installation instructions are [here](https://github.com/richelbilderbeek/djog_unos_2018/blob/develop/doc/install.md)**]

**Comment by same-drenth** [comment]
[click here](https://youtu.be/VWvD4mUpyfU?t=4m) for tutorial

**Comment by richelbilderbeek** [comment]
See [the tutorial video posted by Same](https://youtu.be/VWvD4mUpyfU?t=5m18s) :+1:

**Comment by Joshua260403** [comment]
I did that but the guy opens a txt file and somehow builds an entire project just by doing so. When i tried that it wouldn't work. I'm watching [this](https://www.youtube.com/watch?v=T6zrPBRkURI) tutorial now.

**Comment by Joshua260403** [comment]
I got further, place the sfml source files folder renamed to "SFML" inside of "C:/Qt". The problem now is that i get ALOT of errors.

**Comment by Joshua260403** [comment]
But they are Qt errors, not SFML errors...

**Comment by Joshua260403** [comment]
What Qt version are we using?

**Comment by richelbilderbeek** [comment]
We use Qt5 (can be any subversion number from there). It would be helpful for us if you'd post those errors, possibly in a new Issue, as others may have the same problem.

I have not gotten it to work under Windows yet ...

**Comment by Joshua260403** [comment]
I will continue after school today, i'm not sure if i have Qt5.

**Comment by Joshua260403** [comment]
Whoops

**Comment by Joshua260403** [comment]
I think i might know why same's tutorial didn't work. I found another tutorial ([this one](https://github.com/SFML/SFML/wiki/Tutorial%3A-Compile-and-Link-SFML-with-Qt-Creator)) and here they say you need to install cmake.

**Comment by RafayelGardishyan** [comment]
Joshua, when you're merging a branch I suggest you to fill in the form below the pull request title and also don't merge before travis succeeding (you will se what I men if you open a pull request).
That will help everyone to understand your addition

**Comment by Joshua260403** [comment]
i have never seen such form, and also, every time i merge my branch has already been checked by travis, if it fails (like now) i don't merge

**Comment by Joshua260403** [comment]
Everyone on windows, i believe i succeeded in getting it working on windows. Except for the fact that ubsan doesn't work on windows (#55 ). Please put you're sfml library in a folder called "sfml" in the C:/ directory. I used [this](https://github.com/SFML/SFML/wiki/Tutorial%3A-Compile-and-Link-SFML-with-Qt-Creator) tutorial.

**Comment by Joshua260403** [comment]
oh that thing i a form......

**Comment by richelbilderbeek** [comment]
@Joshua260403: you are the first teenager I know that got SFML working on Windows. Smashingly done! :+1:

(of course, getting it to work in a VM is also cool)

**Comment by Joshua260403** [comment]
too bad travis failed

**Comment by Joshua260403** [comment]
https://travis-ci.org/richelbilderbeek/djog_unos_2018/builds/429607001

**Comment by Joshua260403** [comment]
So is it working now?

**Comment by richelbilderbeek** [comment]
How for are you now?

 * [ ] @annehinrichs22 
 * [ ] @same-drenth 
 * [ ] @martje127 
 * [ ] @Rijk-van-Putten 

Please let me know here if and only if you've succeeded. Without Qt Creator with SFML installed, this project won't be fun.

**Comment by Joshua260403** [comment]
@richelbilderbeek If you want you can schedule me in a second time to show windows users how to do this on windows.

**Comment by Joshua260403** [comment]
Unless there is a problem with windows users using sfml 2.0

**Comment by richelbilderbeek** [comment]
You are scheduled to do so! Also, on Thursdays (19:00-21:00) I could use someone to help the other team. Would you volunteer there as well? No is fine, I just need to know.

**Comment by Joshua260403** [comment]
Thursday i won't be able to come due to a very large quantity of latin
homework

Op wo 19 sep. 2018 07:31 schreef Richel Bilderbeek <notifications@github.com

**Comment by richelbilderbeek** [comment]
Documented [here](https://github.com/richelbilderbeek/djog_unos_2018/blob/develop/doc/install.md)

**Comment by richelbilderbeek** [comment]
Installed Qt Creator and SFML on @martje127's laptop. Thanks @martje127!

**Comment by Joshua260403** [comment]
Latin is done, i'll come to djog today.

**Comment by Joshua260403** [comment]
The current install tutorial isn't how i did it btw

**Comment by Joshua260403** [comment]
I have sfml 2.0 and the tutorial is for 2.5.0 which ia the version that wouldn't work for me at all

**Comment by same-drenth** [comment]
it is all downloaded but not working

**Comment by richelbilderbeek** [comment]
@Rijk-van-Putten let me know by email he got it working. Well done!

**Comment by same-drenth** [comment]
It almost works properly now. 
![screenshot 12](https://user-images.githubusercontent.com/12864136/46017323-7a783500-c0d7-11e8-9803-9392dcecdf37.png)

**Comment by same-drenth** [comment]
Never mind, I got it.

**Comment by richelbilderbeek** [comment]
AFAICS everybody can run the game. Go team!
